README.md

```diff
@@ -33,7 +33,7 @@ MeiliSearch is open-source. You can **support the project by starring** it on [o
 * **Return the whole document**: The entire document is returned upon search.
 * **Highly customizable search and indexation**:
     - [Custom ranking](/guides/main_concepts/relevancy.md): Customize the relevancy of the search engine and the ranking of the search results.
-    - [Stop-words](/guides/advanced_guides/stop_words.md): Ignore common non-relevant words like `of`, `the`, ..
+    - [Stop words](/guides/advanced_guides/stop_words.md): Ignore common non-relevant words like `of`, `the`, ..
```

Comment:
```suggestion
    - [Stop words](/guides/advanced_guides/stop_words.md): Ignore common non-relevant words like `of` or `the`.
```

Reply:
Pretty sure I've already fixed this mistake before. Where does this branch start?